radialgraph/src/main/java/com/duartbreedt/radialgraph/view/RadialGraph.kt

```diff
@@ -136,9 +139,40 @@ class RadialGraph : ConstraintLayout {
         }
     }
 
+    fun setGraphNode(newGraphNodeType: GraphNode) {
+        graphConfig.graphNodeType = newGraphNodeType
+    }
+
+    fun setBackgroundTrack(newColor: Int?) {
+        graphConfig.backgroundTrackDrawable = null
+        graphConfig.backgroundTrackColor = newColor ?: NO_ID
+    }
+
+    fun setBackgroundTrack(newDrawable: Drawable?) {
+        graphConfig.backgroundTrackColor = NO_ID
+        graphConfig.backgroundTrackDrawable = newDrawable
+    }
+
+    fun removeBackgroundTrack() {
+        with (graphConfig) {
+            backgroundTrackColor = NO_ID
+            backgroundTrackDrawable = null
+        }
+    }
+
+    fun redraw(data: Data, animateIn: Boolean = true) {
+        setData(data)
+        if (animateIn) animateIn()
+    }
```

Comment:
I am not sure if this is needed. I'd argue if someone needs to redraw it they should call these functions themselves. 
For instance, what if you want to animateOut first then animateIn the new data? I feel users of this library can extend the graph and add these compound functions in themselves as they are needed.

Reply:
Makes sense. I will remove this from the library then